client/client.go

```diff
@@ -16,6 +18,24 @@ import (
 	"golang.org/x/crypto/ssh/agent"
 )
 
+// InstallPublicFiles installs the public part of the cert and key.
+func InstallPublicFiles(prefix string, cert *ssh.Certificate, pub ssh.PublicKey) error {
+	if prefix == "" {
+		return nil
+	}
+	pubKey := prefix + ".pub"
+	pubCert := prefix + "-cert.pub"
+	pubTxt := ssh.MarshalAuthorizedKey(pub)
+	certPubTxt := []byte(cert.Type() + " " + base64.StdEncoding.EncodeToString(cert.Marshal()))
```

Comment:
Yep. Looked at `strings.Join` for that reason, but it's three strings and it's not in a loop so this way seemed like an easier read.